src/Select/Select.jsx

```diff
@@ -44,15 +45,20 @@ export function Select({
     labelContainerClasses += ` ${cssClass.LABEL_HIDDEN}`;
   }
 
+  let reactSelectClasses = cssClass.REACT_SELECT;
+  if (readOnly) {
+    reactSelectClasses += ` ${cssClass.READ_ONLY}`;
```

Comment:
you can use the `classnames` lib to combine the classes (I think it's already in the repo? if not, this works too.)